res/fm/ecl/ecl_run.py

```diff
@@ -282,9 +283,22 @@ def _get_legacy_run_command(self):
                 self.base_name,
             ]
 
+    def _get_log_name(self, eclrun_config=None):
+        # Eclipse version >= 2019.3 should log to a .OUT file
+        # and not the legacy .LOG file.
+        eclipse_version = (
+            self.sim.version if eclrun_config is None else eclrun_config.version
+        )
+
+        return (
+            "{}.OUT".format(self.base_name)
+            if version.parse(eclipse_version) >= version.parse("2019.3")
```

Comment:
I'm just wandering here is `packaging` module installed here due to `version.parse` function? Is a standard string comparison not working ie.: eclipse_version >= "2019.3". I guess it might be if the ecl version number contains some prefix though

Reply:
Yes, the packaging is included fro the version parse function. I would prefer no to rely on a direct string comparison here, for reasons such as this: 

```
>>> "2019.2" < "2019.20"
True
>>> "2019.2" < "2019.2"
False
```

As mentioned in the PR description, the packaging dependency is already used in a test, and is a requirement of setuptools, so should already be installed in most environments.

But if there are robust alternatives I am all ears!

Reply:
My bad, skipped the PR description :)